trees/python/binary-tree.py

```diff
@@ -0,0 +1,66 @@
+from collections import deque
+
+class Node(object):
```

Comment:
According to [PEP8](https://www.python.org/dev/peps/pep-0008/), there should be a 2 line separation between your import statements and rest of your code.